praw/reddit.py

```diff
@@ -23,12 +23,29 @@
     session,
 )
 
+from typing import (
```

Comment:
As `typing` is part of the standard library, please include it in the topmost section of imports.

Reply:
I must have missed this one